addons/intel/e810.go

```diff
@@ -6,6 +6,12 @@ import (
 	"github.com/openshift/linuxptp-daemon/pkg/plugin"
 	ptpv1 "github.com/openshift/ptp-operator/api/v1"
 	"os/exec"
+	"time"
+)
+
+const (
+	pluginName              = "E810"
+	connectionRetryInterval = 1 * time.Second
```

Comment:
what's this used for?

Reply:
This is not used anymore. I had previously monitored calls here . I have cleaned up